Gemfile

```
@@ -4,7 +4,7 @@ source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby "3.2.2"
-gem "rails", "~> 7.1"
+gem "rails", "7.1.1"
```

Manque d'un commentaire expliquant pourquoi on l'a pin (qu'on puisse facilement juger de si c'est safe ou pas de l'unpin si on oublie dans le futur et qu'on repasse dessus plus tard)

```suggestion
gem "rails", "7.1.1" # Needs https://github.com/hotwired/turbo-rails/issues/512 to be fixed to unpin
```